### PR TITLE
data/snippets : Add code snippet for HAML  

### PR DESCRIPTION
**Pull request by sanket-mali**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by ysahil97** [comment]
Please amend your commit message to follow http://coala.io/commit. Also can you squash both the commits into a single commit?You can refer to [this](http://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits) part in the documentation for the same :smile:

**Comment by ysahil97** [comment]
The commit message needs to follow http://coala.io/commit. Can you please remove the line which is after "Closes ..." and modify the head description of the main commit message? Ping me if you have any doubts. :smile:

**Comment by sanket-mali** [comment]
Review please. I did change the commit message as per requirements..

File: resources/js/snippets.js [hunk 1]
```diff
@@ -14,4 +15,5 @@ var snippets = {
     "Ruby" : "ruby.md",
     "Typescript" : "typescript.md",
     "YAML" : "yaml.md",
+
```
Comment (ysahil97) [inline resources/js/snippets.js]: Can you remove the blank line in line 18?

**Comment by ysahil97** [comment]
In the commit message, can you capitalize 'a' in 'add'?

**Comment by sanket-mali** [comment]
Done.

**Comment by sanket-mali** [comment]
review my changes.

**Comment by hemangsk** [comment]
ack 813a99b

**Pull request merged**